content/actions/using-workflows/workflow-syntax-for-github-actions.md

```diff
@@ -979,12 +979,15 @@ services:
 
 The Docker image to use as the service container to run the action. The value can be the Docker Hub image name or a registry name.
 
+> [!NOTE]
+> You can use the following [AUTOTITLE](/actions/learn-github-actions/contexts) when specifying a service image: `github`, `inputs`, `matrix`, `needs`, `strategy`, and `vars`.
+
 If `jobs.<job_id>.services.<service_id>.image` is assigned an empty string, the service will not start. You can use this to set up conditional services, similar to the following example.
 
 ```yaml
 services:
   nginx:
-    image: {% raw %}${{ options.nginx == true && 'nginx' || '' }}{% endraw %}
+    image: {% raw %}${{ matrix.nginx_service == 'true' && 'nginx' || '' }}{% endraw %}
```

Comment:
Wondering whether `matrix.nginx_service == true` would be better. Booleans are supported matrix values.